File: proxy/typescript/person.ts
// Maske siparişini verecek kullanıcıları temsili sınıf.
export class Person {
  public TC: string;
  public address: string;

  constructor(tc: string, address: string) {
    this.TC = tc;
    this.address = address;
  }
}

File: proxy/go/person.go
package main

// Person maske siparişini verecek kullanıcıları temsili sınıf.
type Person struct {
	tc      string
	address string
}

func NewPerson(tc, address string) Person {
	return Person{
		tc:      tc,
		address: address,
	}
}
